rinja_derive/src/generator.rs

```diff
@@ -124,64 +124,63 @@ impl<'a> Generator<'a> {
         buf.set_discard(false);
 
         self.flush_ws(Ws(None, None));
-        buf.write(CRATE);
-        buf.writeln("::Result::Ok(())");
-        buf.writeln("}");
 
-        buf.writeln(format_args!(
-            "const EXTENSION: ::std::option::Option<&'static ::std::primitive::str> = {:?};",
+        buf.write(format_args!(
+            "\
+                {CRATE}::Result::Ok(()) \
+            }} \
+            const EXTENSION: ::std::option::Option<&'static ::std::primitive::str> = {:?}; \
```

Comment:
I think it's safe to remove whitespace characters at the end of the "line" too. ;)

Reply:
It is, removed. :)